graphqlmetrics/pkg/telemetry/interceptor.go

```diff
@@ -0,0 +1,37 @@
+package telemetry
+
+import (
+	"context"
+
+	"connectrpc.com/connect"
+	"go.opentelemetry.io/otel/attribute"
+)
+
+const (
+	WgOperationName     = attribute.Key("wg.operation.name")
+	WgOperationMethod   = attribute.Key("wg.operation.method")
+	WgOperationProtocol = attribute.Key("wg.operation.protocol")
+)
+
+func defaultAttributes(req connect.AnyRequest) []attribute.KeyValue {
+	var attributes []attribute.KeyValue
+
+	attributes = append(attributes, WgOperationName.String(req.Spec().Procedure))
+	attributes = append(attributes, WgOperationMethod.String(req.HTTPMethod()))
```

Comment:
There are standardized attributes for basic http/net attributes.

Reply:
I updated them to use the GRPC Connect specific one, you can see this in the tests.
I would say this makes sense now.